js/demo-app/src/index.js

```diff
@@ -14,3 +17,15 @@ document.getElementById("router").registerClients({
     assignedRoute: "/two"
   }
 });
+
+// Subscribe to pub-sub events on the topic `publish.topic`
+router.subscribe("publish.topic");
+
+// Listen to publication events (will only be emitted for subscribed topics)
+router.addEventListener("publish", event => {
+  let publication = event.detail;
+  console.log(
+    `Recieved pub-sub data on topic ${publication.topic}:`,
+    publication.payload
+  );
+});
```

Comment:
I'm not sure this is the best API on the host side, but it works for now.  Subscribing to publish events from the `frame-router` element will watch all pub-sub events, but events will only be emitted for topics the host has subscribed to via `subscribe`